platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/EventMetaData.java

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import android.app.ActivityManager;
+import android.content.Context;
+import android.os.Build;
+import android.os.Bundle;
+
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.WindowManager;
+
+import com.google.gson.JsonObject;
+import com.mapbox.mapboxsdk.Mapbox;
+import java.util.Locale;
+
+
+public class EventMetaData {
+
+  private static String METADATA_KEY = "metadata";
+  private static JsonObject eventMetaData = null;
+
+  public static Bundle addMetaData(Bundle data) {
+    if (data == null) {
+      data = new Bundle();
+    }
+    data.putString(METADATA_KEY, getEventMetaData().toString());
+    return data;
+  }
+
+  private static synchronized JsonObject getEventMetaData() {
```

Comment:
metadata should be provided by the caller, this can be removed

Reply:
Removed EventMetaData class.
I thought it could be useful to have a Utility class to do this. 
That kind of Utility can be in the Test.